### DISCUSSION TITLE
MAP ODF issues

### DISCUSSION THREAD

**Discussion by rjvegamcw** [dipy/dipy#2854] [Scientific Visualization]
Hello, I am currently trying to obtain a file with odfs for some volumes in the brain in order to do some tract reconstruction. I am using the processing pipeline highlighted in the tutorial here: [https://dipy.org/documentation/1.7.0/examples_built/07_reconstruction/reconst_mapmri/#sphx-glr-examples-built-07-reconstruction-reconst-mapmri-py](url)

It runs to completion with no errors, however, the image file is merely blank when following the visualization. 
When using the appropriate save and load nifti package, the image saved are merely just intensities, and when using an odf viewer, the error produced is that of: no sh values found. any insight is much appreciated. Happy to expand further if more info is needed. 

My usage is provided below, its pretty much what is used in the tutorial.  
![map usage example](https://github.com/dipy/dipy/assets/139891578/b570e21e-a304-4619-8648-caefb6e4e100)

**Comment by skoudoro** [comment]
Hi @rjvegamcw,

Which visualization tools are you using?  Do you know what your visualization tools expect as input ?

seems related to #2838

I am moving this issue to discussion. it seems to not be a DIPY issue and more a visualization issue.
